src/app/modules/entry-live/views/live-discovery-chart/live-discovery.widget.ts

```diff
@@ -74,43 +74,36 @@ export class LiveDiscoveryWidget extends WidgetBase<LiveDiscoveryData> {
 
   public restoreTimeRange(): void {
     if (this._originalDateRange !== null) {
-      this._dateFilter.startDate = this._originalDateRange.startDate;
-      this._dateFilter.endDate = this._originalDateRange.endDate;
       this._dateFilter.isPresetMode = this._originalDateRange.isPresetMode;
+      if (this._dateFilter.isPresetMode && this._originalDateRange.dateRange) {
+        this._dateFilter.dateRange = this._originalDateRange.dateRange;
+        this._dateFilter.dateRangeServerValue = this._filterService.getDateRangeServerValue(this._originalDateRange.dateRange);
+        console.warn(this._dateFilter.dateRangeServerValue);
```

Comment:
@diamond-darrell - do we need to keep the console output?

Reply:
fixed